azurelinuxagent/common/protocol/wire.py

```diff
@@ -56,6 +56,7 @@
 AGENTS_MANIFEST_FILE_NAME = "{0}.{1}.agentsManifest"
 TRANSPORT_CERT_FILE_NAME = "TransportCert.pem"
 TRANSPORT_PRV_FILE_NAME = "TransportPrivate.pem"
+CONTAINER_ID = "AZURE_GUEST_AGENT_CONTAINER_ID"
```

Comment:
the name of the variable gives me the impression that its value is the container id, try CONTAINER_ID_ENV_VARIABLE or similar

Reply:
let's also document what the variable is used for (basically passing a value across threads)

Reply:
Thanks, updated.